uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -1201,6 +1201,36 @@ sub make_nii {
 }
 
 =pod
+Creates pics associated with MINC files
+=cut
+sub make_minc_pics {
+    my ($dbhr, $TarchiveSource, $profile, $minFileID) = @_;
+    my $where = "WHERE TarchiveSource = ? ";
+    my $query = "SELECT Min(FileID) AS min, Max(FileID) as max FROM files ";
+    $query    = $query . $where;
+    my $sth   = $${dbhr}->prepare($query);
+    $sth->execute($TarchiveSource);
+    print "TarchiveSource is " . $TarchiveSource . "\n";
+
```

Comment:
@gluneau 

I think the print statement was added for debugging purposes (original file did not have it)